### PR TITLE
Fcm buttons issue#147

### PR DESCRIPTION
**Pull request by bullz**
Modified /app/#/models/create buttons

### CODE REVIEW

File: test/protractor-conf.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ exports.config = {
         'browserName': 'chrome'
     },
     baseUrl: 'http://localhost:9000/app/',
-    framework: 'jasmine2',
+    framework: 'jasmine',
```
Comment (FabiApfelkern) [inline test/protractor-conf.js]: Why is this changed?

**Comment by FabiApfelkern** [comment]
There are changes in my tests and test setup? Is this on purpose?

**Comment by mmilaprat** [comment]
I think that is an error. The correct one is your code.

**Comment by FabiApfelkern** [comment]
Can you please change this back!

**Comment by FabiApfelkern** [comment]
@belalakbar Why have you merged this? I have written that there are wrong modifications in my test code.

**Comment by belalakbar** [comment]
its done by mistake, I had reverted it

**Comment by FabiApfelkern** [comment]
But the changes are now in the master.

**Comment by FabiApfelkern** [comment]
I fixed it in the master.

**Comment by belalakbar** [comment]
thanks
